db/db.js

```diff
@@ -37,9 +37,9 @@ const Model = mongoose.model('product', productSchema);
 
 // Model.insertMany(data);
 
-function getAllData() {
+function getData(id) {
```

Comment:
change this query name to be match its new functionality of only grabbing some data and not ALL data like it was before refactoring.